### DISCUSSION TITLE
code-server behind ngnix access react-app-scripts start

### DISCUSSION THREAD

**Discussion by travisboss** [coder/code-server#2471] [General] [answered]
Just thought I'd ask, I think I know the answer but I have my code-server behind a ngnix proxy, specifically linuxserver/swag I am also using there linuxserver/code-server and all is really coming together.

My issue is when I `yarn start` the local is created on `localhost` which does not work for me I need to be able to access on my `domain:3000` instead, is there a variable you all recommend to set in say a dotenv file to change localhost?  I tried changing `HOST` but I get an error about my IP is not available (use split DNS inside my local LAN).

If this is confusing, apologies.

Thanks! Love the product.

**Comment by oxy** [comment, marked as answer]
I think you would need to use code-server's proxy - you can access port 3000 via `http://<code-server-url>/proxy/3000/`.

**Reply by travisboss** [reply]
This is what sort of did it, since i used a subdomain the correct way is `https://3000.code-sever-url.com` for anyone who sees this, also make sure you setup a cname for your host.
